### DISCUSSION TITLE
Admin  user while using HTTP Auth

### DISCUSSION THREAD

**Discussion by xoxys** [FreshRSS/FreshRSS#3690] [Q&A]
Hi,
I'm using FreshRSS with http_auth forwarded to [Authelia](https://github.com/authelia/authelia). While this works great so far, it makes the default admin user not usable anymore, as it is a local user. Is there a way to workaround this?

**Comment by xoxys** [comment]
Maybe the Header Auth in FreshRSS could be extended to allow using more custom header, e.g. to get the email and groups of a user. The groups could be used to set the admin flag to users as well. That's how it's done in Grafana, just for a reference https://grafana.com/docs/grafana/latest/auth/auth-proxy/

**Reply by Alkarex** [reply]
I am not sure I fully understand your question. FreshRSS supports `X-WebAuth-User` like Grafana, but you are asking for something like `X-WebAuth-Groups`, is that right?
What is already possible is that as soon as the users are manually or automatically created in FreshRSS, they can be given the *admin* role.

**Reply by xoxys** [reply]
Sorry, yes I would like to use X-WebAuth-Groups  and apply the FreshRSS admin role to a user that has a customizable group, e.g. `freshrss-admin` in its X-WebAuth-Groups. As FreshRSS users have also an email adress in its profile another variable e.g. X-WebAuth-Email could be used to set this value as well.
